cypress/integration/Landing.spec.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+import testIds from '../../shared/testIds';
+import { getByTestId } from '../utils';
+
+const { landing } = testIds;
+
+describe('[Landing] section', () => {
+  describe('[Hero] CTAs', () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it(`Should go to [/volunteer] when user clicks on [Volunteer]`, () => {
+      cy.get(getByTestId(landing.sections.hero.volunteer)).click();
+      cy.location('pathname').should('eq', '/volunteer');
+    });
+
+    it(`Should go to [/volunteer] when user clicks on [I want to sew masks]`, () => {
+      cy.get(getByTestId(landing.sections.hero.sewMasks)).click();
+      cy.location('pathname').should('eq', '/volunteer');
+    });
+
+    it(`Should go to [/request-supplies] when user clicks on [Get Masks]`, () => {
+      cy.get(getByTestId(landing.sections.hero.requestSupplies)).click();
+      cy.location('pathname').should('eq', '/request-supplies');
+    });
+
+    it(`Should contain reference to [https://rosiesews.freshdesk.com/support/] on [I Can't Sew But I Want To Help]`, () => {
+      cy.get(getByTestId(landing.sections.hero.help))
+        .should('have.attr', 'href')
+        .and('include', 'https://rosiesews.freshdesk.com/support/');
+    });
+  });
+
+  describe('[Get The Pattern] CTAs', () => {
+    Object.keys(landing.sections.afterSignUp.getThePattern).map(pattern =>
+      it(`Should have the pdf associated to [${pattern}]`, () => {
+        cy.get(getByTestId(landing.sections.afterSignUp.getThePattern[pattern]))
+          .should('have.attr', 'href')
+          .and('include', 'pdf');
+      })
+    );
+  });
+
+  describe('[Make Masks] CTAs', () => {
+    const mapPatternRoutes = {
```

Comment:
I believe the pattern pages should be defined via the CMS.  I'm not immediately sure how this could be done, but it would be nice if the pattern tests would update automatically rather than being hard-coded.

Reply:
Yep, totally!

One option could be checking/getting the href and then asserting that it goes there, but if there's a way to get al the registered page templates in the cms that would be amazing